### ISSUE TITLE
'Function' object has no attribute 'get_marker'

### ISSUE THREAD

**Issue by maarcingebala** [celery/celery#5271]
## Checklist

- [x] I have included the output of ``celery -A proj report`` in the issue.
      (if you are not able to do this, then at least specify the Celery
       version affected).
- [x] I have verified that the issue exists against the `master` branch of Celery.

In Pytest 4.1.0 the `get_marker` function was removed, but it is used in Celery 4.2.1. It isn't used anymore on `master` however.

https://docs.pytest.org/en/latest/changelog.html#pytest-4-1-0-2019-01-05

## Steps to reproduce
1. Install `pytest==4.1.0` & `celery==4.2.1`
2. Use `celery_app` fixture in a test.
3. Run tests with `pytest`.

## Expected behavior
`celery_app` fixture works.

## Actual behavior
Tests are failing with the following error:
```
___________________________________ ERROR at setup of test_email_sending_asynchronously ____________________________________
[gw0] darwin -- Python 3.7.0 /Users/marcin/.pyenv/versions/3.7.0/envs/saleor3.7/bin/python

request = <SubRequest 'celery_app' for <Function test_email_sending_asynchronously>>, celery_config = {}
celery_parameters = {}, celery_enable_logging = False, use_celery_app_trap = False

    @pytest.fixture()
    def celery_app(request,
                   celery_config,
                   celery_parameters,
                   celery_enable_logging,
                   use_celery_app_trap):
        """Fixture creating a Celery application instance."""
>       mark = request.node.get_marker('celery')
E       AttributeError: 'Function' object has no attribute 'get_marker'

../../.pyenv/versions/3.7.0/envs/saleor3.7/lib/python3.7/site-packages/celery/contrib/pytest.py:154: AttributeError
```

**Comment by ransford** [comment]
It appears the relevant fix to celery is in #5097. Since pytest is so common, does this merit a minor version bump to 4.2.2?

**Comment by auvipy** [comment]
this should be addressed in 4.3 release soon

**Comment by thedrow** [comment]
Since this is already resolved I'm going to close this.
This enhancement is going to be a part of 4.3.
